client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/PartitionSortedBuffer.java

```diff
@@ -92,7 +90,6 @@ public class PartitionSortedBuffer implements SortBuffer {
   // For writing
   // ---------------------------------------------------------------------------------------------
   /** Whether this sort buffer is released. A released sort buffer can not be used. */
-  @GuardedBy("lock")
```

Comment:
I am assuming this `GuardedBy` is a holdover from some past migration ?
This class is marked as not thread safe, and there is no `lock` instance.

Given this, I removed the annotation